src/odemis/driver/xt_client.py

```diff
@@ -790,6 +790,88 @@ def set_use_case(self, state):
             self.server._pyroClaimOwnership()
             return self.server.set_use_case(state)
 
+    def get_mpp_orientation(self):
+        """
+        Get the current multi probe orientation in degrees.
```

Comment:
Minor comment: could you update this to say "multi probe pattern orientation". Same for the mpp_orientation_info docstring